clients/java/ignored-changes.xml

```diff
@@ -33,4 +33,99 @@
     <method>java.lang.String getGatewayAddress()</method>
     <differenceType>7012</differenceType>
   </difference>
+  <difference>
+    <className>io/zeebe/client/ZeebeClientBuilder</className>
+    <method>io.zeebe.client.ZeebeClientBuilder withJsonMapper(io.zeebe.client.api.JsonMapper)</method>
+    <differenceType>7012</differenceType>
+  </difference>
+  <difference>
+    <className>io/zeebe/client/ZeebeClientConfiguration</className>
+    <method>io.zeebe.client.api.JsonMapper getJsonMapper()</method>
+    <differenceType>7012</differenceType>
+  </difference>
+  <difference>
+    <className>io/zeebe/client/impl/command/CommandWithVariables</className>
```

Comment:
@npepinpe Hi! I don't sure about this. I can't figure out how to change this in a truly backward compatible way :(

Reply:
I don't think it's possible - however this case is probably fine. This class is meant to be subclasses to implement new commands, and it's quite unlikely anyone but us are doing that. I think in this case this is acceptable :+1: